src/redux/Books/bookSlice.jsx

```diff
@@ -30,11 +11,11 @@ const bookSlice = createSlice({
   reducers: {
     addBook: (state, action) => ({
       ...state,
-      book: [...state, action.payload],
+      books: [...state, action.payload],
     }),
```

Comment:
**[OPTIONAL but Highly Recommended]**

- [ ] First of all apologies on behalf of the previous reviewer but it would be very great if you make your addBook method something like below :arrow_down:

```jsx
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },

```
The purpose of doing this :arrow_down:

- Id will be generated automatically while you create a book. Then you are gonna use the same id when you will want to remove any particular book :+1:

Reply:
@HAMMAS-SALEEM, Can you please explain for what exactly you are apologizing, that too for me?